sys-apps/portage/portage-3.0.39-r2.ebuild

```diff
@@ -272,3 +272,12 @@ pkg_preinst() {
 		elog "change, see https://bugs.gentoo.org/658648."
 	fi
 }
+
+pkg_postinst() {
+	# Remove obsolete "enotice" script, bug #867010
+	local bashrc=${EROOT}/etc/portage/profile/profile.bashrc
```

Comment:
Let's move it to a file with a random name (mktemp) as there could be content they want to keep in there?

Reply:
Won't CONFIG_PROTECT come in anyway?

Reply:
Oh wait, postinst.. nevermind

Reply:
Maybe it'd make more sense to just warn about it rather than try to remove it.

Reply:
Changed it to a an eerror message.